src/domains/idpay-app/12_alert.tf

```diff
@@ -17,7 +17,7 @@ resource "azurerm_monitor_action_group" "slackIdpay" {
 resource "azurerm_monitor_scheduled_query_rules_alert_v2" "CompleteOnboarding" {
```

Comment:
Is it normal that var.idpay_alert_enabled is not present inside dev variables? If we don't want it in dev maybe it's better set this variable to false. The plan in dev suggest this
 `azurerm_monitor_action_group.slackIdpay[0] will be destroyed`  because of tha. Could you please add this variable in dev environment ?

Reply:
yes, it's wanted that DEV has not alerts: we let DEV use the default value

You see it already existent because for testing purpose we temporarely changed it to true locally
Now we will destroy them